lib/engine/game/g_1849.rb

```diff
@@ -180,7 +180,7 @@ def setup_companies
       def remove_corp
         removed = @corporations.pop
         @log << "Removed #{removed.name}"
-        return if removed.name == 'AFG'
+        return if removed.name == afg
```

Comment:
shouldn't this just be removed == afg or removed.name == afg.name?

afg is a corp object right

Reply:
oops. fixed.